ex03.exs

```diff
@@ -50,12 +50,29 @@ defmodule Ex03 do
      it is nicely structured            7
   """
 
-  def pmap(collection, process_count, function) do
-    # your code goes here
-    # I'm hoping to see a simple pipeline as the body of this function...
+
+
+def pmap(collection, process_count, function) do
+    #Enum.count(collection)/process_count doesn't work because the stuff you pipe into a function must be the first argument
+    #|> trunc()                               ^I think
+    #|> Enum.chunk_every(collection)
+    Enum.chunk_every(collection, trunc(Enum.count(collection)/process_count)) 
```

Comment:
`trunc(Enum.count(collection)/process_count)` gives the wrong value (try it with 10 and 4, and you get 2, not 3)

# -2